primefaces/src/main/resources/META-INF/primefaces-p.taglib.xml

```diff
@@ -22889,6 +22889,14 @@
             <required>false</required>
             <type>java.lang.String</type>
         </attribute>
+        <attribute>
+            <description>
+                <![CDATA[]]>
```

Comment:
We should add a description here is possible as IntelliJ apparently will ignore fields without a description.